eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/alert/AlertQueueCommand.java

```diff
@@ -0,0 +1,88 @@
+package com.eternalcode.core.feature.essentials.alert;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.notice.NoticeTextType;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.context.Sender;
+import dev.rollczi.litecommands.annotations.join.Join;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.literal.Literal;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import dev.rollczi.litecommands.annotations.command.Command;
+import org.bukkit.entity.Player;
+
+@Command(name = "alert-queue", aliases = { "alertq", "aq" })
```

Comment:
I think aliases should be more consistent with TellRawQueueCommand and also `"aq"` is very unexpected in end-user context.
e.g.
```suggestion
@Command(name = "alert-queue", aliases = { "alert-q" })
```